backend/src/main/java/heartbeat/service/pipeline/buildkite/BuildKiteService.java

```diff
@@ -68,13 +67,12 @@ public BuildKiteResponse fetchPipelineInfo(PipelineParam pipelineParam) {
 					.map(pipeline -> PipelineTransformer.fromBuildKitePipelineDto(pipeline, org.getSlug(),
 							org.getName())))
 				.collect(Collectors.toList());
-			log.info("[BuildKite] Successfully get buildKite pipelineInfo, pipelineInfoList size is:"
-					+ buildKiteInfoList.size());
+			log.info("Successfully get buildKite pipelineInfo, pipelineInfoList size is:" + buildKiteInfoList.size());
 
 			return BuildKiteResponse.builder().pipelineList(buildKiteInfoList).build();
 		}
 		catch (FeignException e) {
-			log.error("[BuildKite] Failed when call BuildKite", e);
+			log.error("Failed when call BuildKite", e);
```

Comment:
update to `Failed to call Buildkite_pipelineParam: {}, e: {}) pipelineParas.xxx, e.message)`

Reply:
done